code/game/objects/items/devices/radio/encryptionkey.dm

```diff
@@ -147,7 +147,7 @@
 /obj/item/encryptionkey/heads/hop
 	name = "\proper the head of personnel's encryption key"
 	icon_state = "cypherkey_cube"
-	channels = list(RADIO_CHANNEL_SERVICE = 1, RADIO_CHANNEL_COMMAND = 1)
+	channels = list(RADIO_CHANNEL_SERVICE = 1, RADIO_CHANNEL_COMMAND = 1, RADIO_CHANNEL_SUPPLY = 1) //monkestation edit: QM isnt a head, HOP should have access to the supply channel.
```

Comment:
Uh, shouldn't this be atomized into it's own PR? This has nothign to do with brig Physician.